tests/Spec/Data/AesonSpec.hs

```diff
@@ -0,0 +1,30 @@
+{-|
+Module:      Spec.Data.AesonSpec
+Copyright:   (C) 2014-2018 Ryan Scott
```

Comment:
You might as well use `(C)` 2022 Steve Mao` here, since (1) we should use the year that this code was first written, and (2) you wrote it.